templates_jinja2/suggest_match_video_playlist.html

```diff
@@ -0,0 +1,91 @@
+{% extends "base.html" %}
+
+{% block title %}The Blue Alliance - Add Match Video{% endblock %}
+
+{% block meta_description %}Add match videos to The Blue Alliance.{% endblock %}
+
+{% block content %}
+<script type="text/javascript">
+    function load_playlist() {
+        var playlist_id = $('#playlist_id').val();
+        var video_list = $('#video_list');
+        video_list.empty();
+        video_list.append($("<p>Loading videos...</p>"));
+        $.get("/_/yt/playlist/videos", {'playlist_id': playlist_id}).done(function(data) {
+            var video_ids = JSON.parse(data);
+            video_list.empty();
+            video_list.append($("<input>", {name: "num_videos", type: "hidden", value: video_ids.length}));
+            $.each(video_ids, function(index, video_data) {
+                var yt_id = video_data['snippet']['resourceId']['videoId'];
+                var li = $("<li>");
+                var group = $('<div>', {class: 'input-group'});
+                li.append(group);
+                group.append($("<a>", {href:"https://youtu.be/"+yt_id}).append($("<h4>"+video_data['snippet']['title']+"</h4>")));
+
+                var match_input = $("<span>");
+                match_input.append("Match Partial: ");
+                match_input.append($("<input>", {name: "video_id_"+index, type: "hidden", value: yt_id}));
+                match_input.append($("<input>", {class: "form-control", name: "match_partial_"+index, type: "text"}));
+                group.append(match_input);
+                video_list.append(li);
+            });
+        });
+    }
+</script>
+
+<div class="container">
+  <div class="row">
+    <div class="col-xs-12 col-lg-6 col-lg-offset-3">
+      {% if num_added %}
+      <div class="alert alert-success">
+        <button type="button" class="close" data-dismiss="alert">&times;</button>
+        <h4>Thanks!</h4>
+        <p>Added {{ num_added }} new suggestions. We'll review them and get them added to the site soon!</p>
+      </div>
+      {% endif %}
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h1 class="panel-title">Help others by adding videos!</h1>
+        </div>
+        <div class="panel-body">
+          <p>Thanks for helping make The Blue Alliance better! Let us know about videos so we can add them to the site!</p>
+          <ul>
+            <li>Your suggestion will be reviewed by a moderator</li>
+            <li>Your account info (like {{user_bundle.user.email()}} and {{user_bundle.user.nickname()}}) will also be submitted. <a href="{{user_bundle.logout_url}}">log in as a different user</a></li>
+          </ul>
+          <div class="alert alert-success">
+            <strong>Match Partial Examples</strong>
```

Comment:
I think we should call this "Match ID" everywhere user-facing. It's more digestible than "Match Partial" IMO